src/clib/ct.cpp

```diff
@@ -1420,6 +1420,17 @@ extern "C" {
         }
     }
 
+    int getDataDirectories(int buflen, char* buf, const char* sep)
```

Comment:
I think the function should be named `ct_getDataDirectories` (some of the existing functions need to be updated to follow this convention). Also, I think you forgot to add `ct.h`.